app/assets/javascripts/actions/story.js

```diff
@@ -10,6 +10,24 @@ export const toggleStory = (id) => ({
   id
 });
 
+export const updateStorySuccess = (story) => ({
+  type: actionTypes.UPDATE_STORY_SUCCESS,
+  story
+})
+
+export const updateStory = (story, projectId) => {
+  return (dispatch , getState, { Story }) => {
+    if (story._editing._isDirty) {
+      return Story.update(story._editing, projectId)
+        .then(( story ) => {
+          dispatch(updateStorySuccess(story));
+          dispatch(toggleStory(story.id));
```

Comment:
_What if_ we add a `case` [in the reducer](https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/reducers/stories.js#L10) to toggle the story when `UPDATE_STORY_SUCCESS` happens as well so we don't have to trigger two actions here? It'll make we save some processing time by not rendering the story twice.